src/components/sidebar/routePathView/NewRoutePathView.tsx

```diff
@@ -88,40 +108,45 @@ class NewRoutePathView extends React.Component<INewRoutePathViewProps, INewRoute
     }
 
     public onChange = (property: string, value: string) => {
-        this.setState({
-            currentRoutePath: { ...this.state.currentRoutePath!, [property]: value },
-        });
+        this.props.routePathStore!.updateRoutePathField(property, value);
```

Comment:
updateRoutePathProperty, (we pass property named variable there) + property is used in our project rather than field